cblite/EnrichCommand.hh

```diff
@@ -30,4 +30,6 @@ public:
     void runSubcommand() override;
 protected:
```

Comment:
Is there a need for this to be protected vs private?

Reply:
Not really, I just made it protected because that's how the other commands have it. I can change it if needed.

Reply:
It looks like the other instances are because those classes are inherited by something else.  This is not, and should be private.  This is probably the last small thing.